Packs/UrlScan/Integrations/UrlScan/UrlScan.py

```diff
@@ -503,22 +546,42 @@ def get_urlscan_submit_results_polling(client, uuid):
 
 
 def urlscan_submit_command(client):
+    execution_metrics = ExecutionMetrics()
+    command_results: list = []
+    items_to_schedule: list = []
+    rate_limit_reset_after: int = 60
+
     urls = argToList(demisto.args().get('url'))
     for url in urls:
         demisto.args()['url'] = url
-        response = urlscan_submit_url(client)
-        if response.get('url_is_blacklisted'):
-            pass
+        response, metrics, rate_limit_reset_after = urlscan_submit_url(client, url)
+        if response.get('url_is_blacklisted') or response.get('is_error'):
+            execution_metrics.general_error += 1
+            if response.get('is_error'):
+                schedule_and_report(command_results=command_results, items_to_schedule=items_to_schedule,
+                                    execution_metrics=execution_metrics, rate_limit_reset_after=rate_limit_reset_after)
+                return_results(results=command_results)
+                return_error(response.get('error_string'))
+            continue
+        if metrics == ErrorTypes.QUOTA_ERROR:
+            if not is_scheduled_command_retry():
+                execution_metrics.quota_error += 1
+            items_to_schedule.append(url)
+            continue
         uuid = response.get('uuid')
         get_urlscan_submit_results_polling(client, uuid)
+        execution_metrics.success += 1
+    schedule_and_report(command_results=command_results, items_to_schedule=items_to_schedule,
```

Comment:
please run a test(not unit test) where we get an quota error and it is bing handled by this successfully.